examples/CSharp/Tutorial 1.5 - Transformations/Transform.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using System.Numerics;
+using System.Text;
+
+namespace Tutorial
+{
+    public class Transform
```

Comment:
Could this be converted to a `struct`? It feels kinda icky using a class for data such as this.

Reply:
Problem with this is that means we cant use field initialisers which are essential as you might not want to set a rotation for an object. This could maybe be elaboratedd on in the web version saying that people could use structs but we use classes for this reason

Reply:
Hmm fair enough. Reluctantly letting this slide :P